aiokinesis/producer.py

```diff
@@ -55,7 +55,7 @@ def _complete_produce_request(self, task):
                 loop=self._loop
             )
             task.add_done_callback(
-                partial(self._complete_produce_request)
+                partial(self._complete_produce_request, task)
```

Comment:
I don't think this is right... 

the description for this method in asyncio library is 
```
    def add_done_callback(self, *args, **kwargs): # real signature unknown
        """
        Add a callback to be run when the future becomes done.
        
        The callback is called with a single argument - the future object. If
        the future is already done when this is called, the callback is
        scheduled with call_soon.
        """
        pass
```

Reply:
the future object is `task`, as such it's already being passed in and we shouldn't have to do it explicitly